src/lib.rs

```diff
@@ -58,18 +66,20 @@ impl<W, T> PistonWindow<W, T>
         use piston::event::Events;
         use piston::window::{ OpenGLWindow, Window };
 
-        let (mut device, mut factory) = gfx_device_gl::create(|s| window.borrow_mut().get_proc_address(s));
+        let mut device = gfx_device_gl::Device::new(|s| window.borrow_mut().get_proc_address(s));
+        let mut factory = device.spawn_factory();
```

Comment:
Hmmm I thought that adding factory as a member of `PistonWindow` might then make it available for use with gfx_graphics' [`GlyphCache::new`](https://github.com/PistonDevelopers/gfx_graphics/blob/master/src/glyph.rs#L44) and [`GlyphCache::update`](https://github.com/PistonDevelopers/gfx_graphics/blob/master/src/glyph.rs#L58) method, but it seems that even if I add it, I still get the following error when trying to use it:
```
error: the trait `gfx::device::Factory<_>` is not implemented for the type `&mut gfx_device_gl::factory::Factory`
let mut glyph_cache = GlyphCache::new(&font_path, &mut window.factory.borrow_mut().deref_mut()).unwrap();
```
Perhaps the gfx_graphics and gfx_texture APIs need some changes to catch up with the latest gfx-rs version?